src/Plutarch/Extra/Map.hs

```diff
@@ -96,3 +107,71 @@ pkeys = phoistAcyclic $
     plam $ \m -> unTermCont $ do
         PMap kvs <- pmatchC m
         pure $ pfmap # pfstBuiltin # kvs
+
+{- | / O(n) /. Update the value at a given key in a `PMap`, have the same functionalities as 'Data.Map.update'.
+
+     @since 1.0.0
```

Comment:
We should probably bump the version if we're adding things. Specifically bump it to 1.1.0.